### PR TITLE
refactor: seo title & description asset hub

### LINKED ISSUE #7645: explorer seo tune for Asset Hub
I would like to see included in the title and description both. 

https://kodadot.xyz/ahk/explore/items - Items on Asset Hub Kusama 
https://kodadot.xyz/ahp/explore/items - Items on Asset Hub Polkadot
https://kodadot.xyz/ahk/explore/collectibles - Collections on Asset Hub Kusama 
https://kodadot.xyz/ahp/explore/collectibles - Collections on Asset Hub Polkadot

Other for h1 text, can we extend it with "Explore Items on Asset Hub Polkadot" / "Explore Items on Asset Hub Kusama" accordingly 

image

### PR DESCRIPTION
**Pull request by thnaylor**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [ ] Closes #7645 
- [ ] Requires deployment <snek/rubick/worker>

#### Did your issue had any of the "$" label on it?

- [ ] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=123PCJXhjb15i6JwVVRGd7KvN2sSNZrtPjr5doJq9oWctoTt)

## Screenshots

image

image

image

image

image

image

### <samp>🤖 Generated by Copilot at ba92bed</samp>

This pull request adds dynamic SEO meta tags and page titles to the explore pages based on the URL prefix and the chain or asset hub. It uses a new utility file `seo.ts` and updates the explore layout and pages scripts and the English locale file.

### <samp>🤖 Generated by Copilot at ba92bed</samp>

> _We are the masters of SEO_
> _We optimize our pages with `seo.ts`_
> _We use the prefix and the hub to set the meta tags_
> _We make our titles dynamic and localized_

### CODE REVIEW

**Comment by exezbcz** [comment]
does not look that good now, but if it helps with seo..

I would maybe leave out the "items", because this is not logical no? There should be either "collections" or nothing 
image

**Comment by yangwao** [comment]
> but if it helps with SEO..

hope so for bit better rank as it's h1 text (high value)

> I would maybe leave out the "items", because this is not logical no? There should be either "collections" or nothing

yup, [it's in the highlight](https://github.com/kodadot/nft-gallery/issues/7645#:~:text=kodadot.xyz/ahk/explore/collectibles%20%2D%20Collections%20on%20Asset%20Hub%20Kusama%0Akodadot.xyz/ahp/explore/collectibles%20%2D%20Collections%20on%20Asset%20Hub%20Polkadot)

File: pages/[prefix]/explore/collectibles.vue [hunk 1]
```diff
@@ -15,6 +16,22 @@ const checkRouteAvailability = () => {
...
 }
 
+const getSeoMeta = computed(() => {
+  const prefix = urlPrefix.value
+  const isAssetHub =
+    Object.keys(chainNameSeoMap).includes(prefix) && assetHub.includes(prefix)
+
+  return {
+    title: isAssetHub
+      ? `Collections on Asset Hub ${getSeoPrefixName(prefix)}`
```
Comment (yangwao) [inline pages/[prefix]/explore/collectibles.vue:26]: seems this one doesn't work?

image

**Review by yangwao — CHANGES_REQUESTED** [review]

File: pages/[prefix]/explore/collectibles.vue (hunk 1 above)
Comment (yangwao) [reply pages/[prefix]/explore/collectibles.vue:26]: yes, yet not sure how crawler bots will see it, let's give it shot

**Comment by yangwao** [comment]
interesting second time failed deploy 
image

**Comment by yangwao** [comment]
pay 30 usd

let's go, let's see if will have some impact on seo

image

**Comment by yangwao** [comment]
thanks @thnaylor

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 3.78 USD/DOT ~ 7.937 $DOT
🧗 [123PCJXhjb15i6JwVVRGd7KvN2sSNZrtPjr5doJq9oWctoTt](https://polkadot.subscan.io/account/123PCJXhjb15i6JwVVRGd7KvN2sSNZrtPjr5doJq9oWctoTt)
🔗 [0x5f1c34cb6b56778385d9f7225fa67e1476ff6cb50a8fd32885d81e85131ff693](https://polkadot.subscan.io/extrinsic/0x5f1c34cb6b56778385d9f7225fa67e1476ff6cb50a8fd32885d81e85131ff693)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**